components/eam/src/chemistry/mozart/mo_usrrxt.F90

```diff
@@ -298,283 +298,46 @@ subroutine usrrxt_inti
 
   end subroutine usrrxt_inti
 
-  subroutine usrrxt( rxt, temp, tempi, tempe, invariants, h2ovmr,  ps, &
-                     pmid, m, sulfate, mmr, relhum, strato_sad, &
-                     ltrop, ncol, sad_total, cwat, mbar, pbuf )
+  subroutine usrrxt( rxt, temp, invariants, mtot, ncol )
```

Comment:
I have a more general question regarding this comment: do we want to refactor (e.g., remove unused indices) the *_init subroutines? Previously we just skip all the initialization subroutines.

Reply:
That's a good point. If we remove the module-level variables, we will have to remove those from the init routines. We can leave them as is but then it may look like they are being used somewhere in the code. This code is quite simple, so I thought removing them will make it look even simpler.

As it doesn't matter, @brigaudet, please feel free to keep them if it is too much work.

Reply:
I did go ahead, and removed almost all of these both at the module level and in the initialization subroutine; the majority is not used according to both the code coverage tool and the documentation within pp_linoz_mam4_resus_mom_soag.  I did retain field definitions which are invoked in the write iulog statement at the end of the initialization subroutine.  I also removed some more unused code in other chemistry files that I am currently or previously refactored.